README.md

```diff
@@ -55,7 +55,7 @@ Compatibility
 
 * [ember-concurrency][e-c] 2.0.0 or above
 * [ember-concurrency-async][e-c-async] 0.2.0 or above (optional)
-* Ember.js v3.12 or above
+* Ember.js v3.16 or above
 * Ember CLI v2.13 or above
 * Node.js v10 or above
```

Comment:
If we need to do a breaking change, it’s a good opportunity to drop node 10

Reply:
@chancancode I updated `package.json` but missed README.md. Fixed! Thanks for catching!